packages/jwt/src/ByuJwt.ts

```diff
@@ -0,0 +1,103 @@
+import NodeCache from 'node-cache'
+import {
+  OpenIdConfiguration,
+  Certifications,
```

Comment:
❗ These are _certificates_, not certifications. If you want to use the terminology from OIDC, we're talking about JWKS or "JSON Web Key Sets."

@stuft2 -This would be a blocking concern if it were part of our public contract. Since we're not exporting anything with a `Certifications` type, I'm okay if this gets fixed as part of a follow-up PR.

Reply:
fixed ✅